ceph/rbd/workflows/rbd_mirror.py

```diff
@@ -561,6 +561,47 @@ def config_mirror_multi_pool(
     return output
 
 
+def check_pool_mirror_status(mirror_obj, pool_name, cluster_name, status, timeout=120):
```

Comment:
Name changed to compare_pool_mirror_status would give more meaning I feel.

Also in below description on Line 566,

You can write, check if the pool mirror status is the same as the value passed in status argument.

Reply:
Done